src/pages/_components/Sponsors.astro

```diff
@@ -57,6 +57,15 @@ const mainSponsorLogoHeight = 40
 				height={mainSponsorLogoHeight}
 			/>
 		</MainSponsorItem>
+		<MainSponsorItem href="https://sentry.io/" caption="Official Bug Smashing Partner">
```

Comment:
Is this intentional lmao

Reply:
Haha, no I just missed committing the updated caption!

Though "Official Bug Smash Partner" has a nice ring to it

Reply:
Haha, no I just missed committing the updated caption!

Though "Official Bug Smash Partner" has a nice ring to it...